v0/destinations/heap/transform.js

```diff
@@ -7,7 +7,51 @@ const {
   defaultRequestConfig
 } = require("../util");
 
-function responseBuilder(payload, message, heapConfig) {
+function flatten(target, opts) {
```

Comment:
heap needs  flattened properties?
we are not using opts, can this be removed? Also, move this method to utils?
can we have a description and an example of what it does with javascript object, object array and array types

Reply:
this function is also used in another destination Kissmetrics. there opts is used so making it generic and keeping it in utils also adding comments.

Reply:
Yes heap was not accepting nested properties.